.github/workflows/test-workflow.yml

```diff
@@ -0,0 +1,67 @@
+name: Test Frontend and backend
+# This workflow is triggered on pushes to the repository.
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
No not really a reason. I can do it for all pull_requests.

Reply:
@jelhan I found the issue: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#pull-request-events-for-forked-repositories

so you have to allow it.

Reply:
Are you sure? For me that sounds more like:

- If you fork a repo, workflows won't run on your fork automatically. You need to enable that explicitly.
- If you create a pull request from a fork repository, the `pull_request` event will be emitted to the base repository. It will run the CI pipeline as configured by the base repository.

If I got it right, that's also the reason why your pull request does not trigger a GitHub action: this repo does not have a workflow yet for pull requests. It will have one as soon as your merge request is merged. So I think not seeing a GitHub action triggered by your pull request is fine.

I'm using the GitHub actions on your fork to see how the CI pipeline looks in practice.